### PR TITLE
catalog grid

### PR DESCRIPTION
**Pull request by IvanTkachenko3301**
[DEMO LINK](https://IvanTkachenko3301.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by Kavon-Skela — APPROVED** [review]
Good job overall, but you need to fix a few mistakes with media queries. It is not critical, but it is not according to the checklist

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+  .catalog {
+    grid-template-columns: repeat(2, $card-size);
   }
+}
+
+@media (min-width: 769px) {
+  .catalog {
+    grid-template-columns: repeat(3, $card-size);
+  }
+}
```
Comment (Kavon-Skela) [inline src/styles/_catalog.scss:33]: This part of code needs to be changed

**Review by Kavon-Skela — APPROVED** [review]

File: .linthtmlrc.json [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+  "tag-name-lowercase": true,
+  "tag-name-match": true,
+  "tag-self-close": "never",
+  "tag-close": true,
+  "text-ignore-regex": "&",
+  "title-no-dup": true,
+  "line-end-style": "lf",
+  "attr-new-line": 2,
+  "attr-name-style": "dash",
+  "attr-no-unsafe-char": true
```
Comment (SerhiiKirik) [inline .linthtmlrc.json:48]: On the feature - Do not add extra files in PR
To add the file to a commit use git add src/.

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,22 +1,39 @@
...
     align-items: center;
-    width: 200px;
+    width: $card-size;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    &:last-child {
+      margin-bottom: 48px;
+    }
```
Comment (SerhiiKirik) [inline src/styles/_catalog.scss:19]: it's extra
```suggestion
```

File: src/styles/_catalog.scss [hunk 4]
```diff
@@ -1,22 +1,39 @@
...
 
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
+  justify-content: center;
+  gap: 48px;
+
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
```
Comment (SerhiiKirik) [inline src/styles/_catalog.scss:12]:
```suggestion
    display: grid;
    justify-content: center;
    align-items: center;
```

File: src/styles/_catalog.scss [hunk 5]
```diff
@@ -1,22 +1,39 @@
...
+    grid-template-columns: repeat(3, $card-size);
+  }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, $card-size);
   }
```
Comment (SerhiiKirik) [inline src/styles/_catalog.scss:38]: move this media to catalog class

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🚀 
Let's fix some problems & improve your code 🧑‍💻 
image

image
image

**Review by VladislavaT — APPROVED** [review]

File: src/styles/_catalog.scss [hunk 6]
```diff
@@ -1,22 +1,36 @@
...
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, $card-size);
+    }
+  }
+
   &__card {
-    display: flex;
+    display: grid;
```
Comment (jstmpelowycz) [inline src/styles/_catalog.scss:28]: you don't need to apply grid to cards

File: src/styles/_catalog.scss [hunk 7]
```diff
@@ -1,22 +1,36 @@
...
+    & {
+      grid-template-columns: repeat(3, $card-size);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, $card-size);
+    }
+  }
```
Comment (jstmpelowycz) [inline src/styles/_catalog.scss:25]: you don't need `& {}` syntax

File: src/styles/_catalog.scss [hunk 8]
```diff
@@ -1,22 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
   margin: 0 auto;
 
+  display: grid;
+  grid-template-columns: repeat(1, $card-size);
```
Comment (jstmpelowycz) [inline src/styles/_catalog.scss:5]: not required, but OK

File: .linthtmlrc.json (hunk 2 above)
Comment (jstmpelowycz) [reply .linthtmlrc.json:48]: great point

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

File: src/styles/_catalog.scss [hunk 9]
```diff
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
   margin: 0 auto;
```
Comment (liliia-livandovskaya) [inline src/styles/_catalog.scss:2]:
```suggestion
```

**Review by liliia-livandovskaya — APPROVED** [review]
Good job! 👍

**Pull request closed without merging**